tendrl/node_agent/objects/definition/node_agent.yaml

```diff
@@ -15,6 +15,25 @@ namespace.node_agent:
       uuid: 3f94a48a-05d7-408c-b400-e27827f4eacc
       type: Create
       version: 1
+    ConfigureCollectd:
+      atoms:
+        - tendrl.objects.Node.atoms.Cmd
+      help: "Execute given command on given node"
+      enabled: true
+      inputs:
+        mandatory:
+          - 'Node[]'
+          - Service.name
+          - graphite_host
```

Comment:
this is not how inputs work. Isnt service.name always going to be the same?  also, graphite_host and graphite_port arent linked to any objects.

I think we can hardcode the service name, and use graphite_host and graphite_port directly from node-agent conf